.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ env:
   MIX_ENV: test
   otp-version: '24'
   elixir-version: '1.12'
-  cache-version: '0'
+  cache-version: '1'
```

Comment:
I needed to bust the cache per an  "Old PLT found" error. If we  keep seeing this, we'll need to adjust caching of dialyzer PLTs. First time I've encountered this one.